gsoc/anand/PIPELINE.md

```diff
@@ -4,18 +4,18 @@ Follow the below given steps
 
 ## Steps to follow
 
-### One - liner
+### One - liner (For Step 1-6)
 
 Here is the skeleton of the command:
 
 ``` bash
-python final_formatting.py --rs <0 IF TO RUN THE WHOLE CODE | 1 TO RUN THE FUNTION OF THIS PYHTON SOURCE ONLY> --uri_file <URI FREQUENCY FILE TSV> --url <WEBPAGE URL> --output_file <OUTPUT FILE NAME > --project_name <PROJECT NAME> --namespace <NAMESPACE>
+python3 final_formatting.py --rs <0 IF TO RUN THE WHOLE CODE | 1 TO RUN THE FUNTION OF THIS PYHTON SOURCE ONLY> --uri_file <URI FREQUENCY FILE TSV> --url <WEBPAGE URL> --output_file <OUTPUT FILE NAME > --project_name <PROJECT NAME> --namespace <NAMESPACE>
```

Comment:
I would suggest you to keep it as `python` and add points in the description instructing the user to use python version 3 (or a specific version) while running the code.

Reply:
Done.